### ISSUE TITLE
`const const const const` makes a variable unusable

### ISSUE THREAD

**Issue by MrDiamondDog** [TodePond/DreamBerd#256]
`const const const const` should make a variable unusable. Completely useless.

```
const const const const name = "world"!
print(name)! // Error: can not read const const const const variable.
```

**Comment by Iuriiiiiii** [comment]
I like it, but i think that if you put a "const" more there, you could make unusable that variable on all your projects

**Comment by MrDiamondDog** [comment]
So `const const const const const` makes it completely unusable throughout your entire computer? I like it

**Comment by jerbear2008** [comment]
Don't forget that `const const const const const const` makes it constant, immutable and, unreadable on all devices worldwide forever. (For those of you at home, that is 6 `const`s.)

**Comment by MrDiamondDog** [comment]
And you can’t forgot `const const const const const const const`, which makes it unusable throughout the entire galaxy.

**Comment by FKCCF** [comment]
> So `const const const const const` makes it completely unusable throughout your entire computer? I like it

No!It should make 'const' unusable,so u can only use 'var' later!

**Comment by MrDiamondDog** [comment]
That's what `delete const` is for

**Comment by stohrendorf** [comment]
> And you can’t forgot `const const const const const const const`, which makes it unusable throughout the entire galaxy.

This could be combined with #119 and its proposed `universe` to something like `universe delete const`.

**Comment by ktlknss** [comment]
You can't forget `const const const const var var const const const const const var var var var var var var var var var var const const` i wish i had this in DB it would be so useful
